fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -200,8 +207,9 @@ public void addLogicalResources(List<IDatabaseObject> group, String prefix) {
         // We also have a FK constraint pointing back to that table to try and keep
         // things sensible.
         Table.Builder builder = Table.builder(schemaName, tableName)
+                .setVersion(FhirSchemaVersion.V0027.vid()) // V0027: add support for distribution/sharding
                 .setTenantColumnName(MT_ID)
-                .setVersion(FhirSchemaVersion.V0020.vid()) // V0020: Updated to support Postgres fillfactor changes
+                .setDistributionType(DistributionType.DISTRIBUTED) // V0027 support for sharding
```

Comment:
which column are these search parameter tables distributed on?

Reply:
I think I found the answer in ShardedSchemaAdapter.java ... there is a special column that is automatically added

Reply:
Yes, the schema adapter contains the default distribution column name. Ignore the ShardedSchemaAdapter for now. For Citus we'll be using the DistributedSchemaAdapter, which is constructed with the default distribution column name we intend to use (LOGICAL_RESOURCE_ID).